### PR TITLE
feat(inputs): variable for input opacity

### PR DESCRIPTION
**Pull request by CarloBar1**
Feature [AB#11519](https://dev.azure.com/neovici/7e94365d-32b1-416f-854b-7f195da31da6/_workitems/edit/11519) - styling tweaks. The opacity of the inputs is now settable from the outside and the "Required" error message is no longer shown
image

### CODE REVIEW

File: src/styles.ts [hunk 1]
```diff
@@ -42,8 +42,8 @@ export const styles = css`
 		padding-top: var(--paper-input-container_-_padding-top, 8px);
 		padding-bottom: var(--paper-input-container_-_padding-bottom, 8px);
 		position: relative;
-		transition:
-			transform 0.25s,
+		transition: 
+			transform 0.25s,
```
Comment (cristinecula) [inline src/styles.ts:46]: Remove the unnecessary whitespace.

File: src/styles.ts [hunk 2]
```diff
@@ -100,8 +101,8 @@ export const styles = css`
 		top: 0;
 		left: 0;
 		width: var(--cosmoz-input-label-width, 100%);
-		transition:
-			transform 0.25s,
+		transition: 
+			transform 0.25s,
```
Comment (cristinecula) [inline src/styles.ts:105]: Remove the unnecessary whitespace.

File: stories/cosmoz-input.stories.js [hunk 3]
```diff
@@ -53,6 +53,7 @@ export const contour = () => html`
 			--cosmoz-input-line-display: none;
 			--cosmoz-input-contour-size: 1px;
 			--cosmoz-input-background: white;
+			--cosmoz-input-opacity: 0.7;
```
Comment (cristinecula) [inline stories/cosmoz-input.stories.js:56]: I don't think we want to have the inputs featured in this story be semi-transparent.

**Review by cristinecula — CHANGES_REQUESTED** [review]
The commit message `feat: css var for input opacity` would be better.

File: stories/cosmoz-input.stories.js (hunk 3 above)
Comment (CarloBar1) [reply stories/cosmoz-input.stories.js:56]: Yes, but this is the whole point of the change. According to [mockups](https://www.figma.com/file/eczucze3bcnenVmnPDnbrK/action-dialog?type=design&node-id=0-1&mode=design&t=SAe332WbgGfLQmzu-0) and the UX meetings, the inputs in the forms should all be in a lighter color. 

I thought of setting the color variable directly, but the risk I foresee is that the disabled input characteristics might not be obvious anymore, so I went for a general layer of opacity

Comment (cristinecula) [reply stories/cosmoz-input.stories.js]: Opacity is not the correct way to make things lighter. You have to set each color to a lighter, solid color. You should never use opacity in this way. We can merge this PR as it might come useful in certain situations, but not for this purpose.

**Review by cristinecula — APPROVED** [review]
